addons/cookoff/stringtable.xml

```diff
@@ -7,6 +7,7 @@
             <Chinese>殉爆效果</Chinese>
             <Chinesesimp>殉爆效果</Chinesesimp>
             <Japanese>誘爆</Japanese>
+            <German>Selbstzündung</German>
```

Comment:
Ich finde die Übersetzung komisch. Zumindest laut [deutscher Wikipedia](https://de.wikipedia.org/wiki/Cook_off) nutzt man im Deutschen ebenfalls das Wort "Cook off".

Alternativ fände ich "Munitionsbrand" ganz nett.

Reply:
Nennt man "Durchzündung".

Reply:
Selbstzündung war wenn ich mich recht erinnere der Begriff der an anderer Stelle in der Stringtable verwendet wurde. Ich ändere es jetzt mal alles zu Durchzündung. Munitionsbrand finde ich auch ganz gut. Was ihr verwenden wollt müsst dann ihr entscheiden :)